src/pages/ResultsPage.js

```diff
@@ -22,11 +22,11 @@ export default function ResultsPage({ onBack, onDetails }) {
       </SpinnerWrapper>
     )
   return (
-    <Animation>
-      <Test>
+    <Wrapper>
+      <Top>
         <Header>Search for '{searchTerm}'</Header>
         <BackButton onClick={onBack}>X</BackButton>
```

Comment:
this one might also be a link. maybe think about creating a link component since you seem to use it quite a bit